tsdapiclient/tacl.py

```diff
@@ -485,6 +500,11 @@ def construct_correct_upload_path(path: str) -> str:
     default=CHUNK_THRESHOLD,
     help='E.g.: 1gb, files larger than this size will be sent as resumable uploads'
 )
+@click.option(
+    '--remote-path',
+    required=False,
+    help='Specify a path on the remote server to upload'
```

Comment:
Isn't it for both upload and download?

Reply:
Yes have to update the strings and guide